dashboard/src/types.ts

```diff
@@ -0,0 +1,25 @@
+export interface MetricPoint {
```

Comment:
I think one file per type would be better
or at the very least one file for type group (all metric related in one all graph related on another)